selfdrive/locationd/locationd.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, debug: bool):
 
     self.debug = debug
 
-    self.posenet_stds = [POSENET_STD_INITIAL_VALUE] * (POSENET_STD_HIST_HALF * 2)
+    self.posenet_stds = np.array([POSENET_STD_INITIAL_VALUE] * (POSENET_STD_HIST_HALF * 2))
```

Comment:
is a `deque` faster here?

Reply:
deque does not support slicing, so it cannot be used directly with :

```
old_mean = np.mean(self.posenet_stds[:POSENET_STD_HIST_HALF])
new_mean = np.mean(self.posenet_stds[POSENET_STD_HIST_HALF:])
```
Additionally, using np.array for posenet_stds can improve np.mean performance because passing a python list to np.mean requires numpy to internally convert the list to a no.array.